nixui/options/api.py

```diff
@@ -18,20 +18,14 @@ class NoDefaultSet:
 @copy_decorator.return_copy
 @functools.lru_cache(1)
 def get_release_json():
-    release_path = os.path.join(store.get_store_path(), 'release_out')
-
-    # TODO - fix hack: release will change and this needs to be reflected, the parser should parse nixpkgs either each run or each time it changes
-    if not os.path.exists(release_path):
-        subprocess.run([
-            'nix-build',
-            '<nixpkgs/nixos/release.nix>',
-            '-A',
-            'options',
-            '-o',
-            release_path
-        ])
-    release_options_json_path = os.path.join(release_path, 'share', 'doc', 'nixos', 'options.json')
-    return json.load(open(release_options_json_path))
+    return nix_eval.nix_instantiate_eval("""
+        with import <nixpkgs/nixos> {};
+        builtins.mapAttrs
+           (n: v: builtins.removeAttrs v ["default" "declarations"])
```

Comment:
Why remove `default`?

Also, should we iterate over these and /attempt/ to get `unsafeGetAttrPos` metadata?

Reply:
Or perhaps that can be a PR that follows?

Reply:
When calling `nixosOptionsDoc` directly, the `default` value can contain actual packages from `nixpkgs`, that is, it refers to actual `/nix/store/...` paths, and also require those paths to be realized. `nixpkgs/nixos` implements the workaround here. https://github.com/NixOS/nixpkgs/blob/26f64b251cbac73569e2f5cf83b27f1b0f1fe011/nixos/modules/misc/documentation.nix#L11-L41

I find this non-trivial so I did not attempt to re-implement it on this PR.

Reply:
Ok, we can skip that, we may not even care about the default.

Could you mention which attributes are included in a docstring for the function?